ui/src/app/segments/Segment.tsx

```diff
@@ -342,7 +342,7 @@ export default function Segment() {
                         <td className="text-gray-500 hidden whitespace-nowrap px-3 py-4 text-sm lg:table-cell">
                           {ConstraintOperators[constraint.operator]}
                         </td>
-                        <td className="text-gray-500 hidden whitespace-nowrap px-3 py-4 text-sm lg:table-cell">
+                        <td className="text-gray-500 hidden whitespace-normal px-3 py-4 text-sm lg:table-cell">
```

Comment:
The constraints table doesn't play well with a large values and since json lists will be pretty big, I have added text wrapping for the value column.

I think the value column should be collapsible for list content i.e., only show n values as chips first and show all values on click. I can create another PR for that change if it sounds alright.

Reply:
yeah I like that idea. I also wonder if we could use [truncate](https://tailwindcss.com/docs/text-overflow#truncate)?

Reply:
I tried using `truncate` but doesn't seem to be working here trivially.